### PR TITLE
Remove support for address, contact info from PR API (align with pull request 955)

### PR DESCRIPTION
**Pull request by ianbjacobs**
1) Andre Lyver moved to former editor
2) Removed shipping, contact information to align with PR 955 on PR API
   https://github.com/w3c/payment-request/pull/955

The following tasks have been completed:

 * [x] tidy
 * [x] no respect errors
 * [ ] web platform tests (link)
 * [ ] MDN Docs added (link)

Implementation commitment:

 * [ ] Safari (link to issue)
 * [ ] [Chrome](https://crbug.com/1207601)
 * [ ] Firefox (link to issue)
 * [ ] Edge (public signal)

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/payment-handler/pull/389.html" title="Last updated on May 10, 2021, 8:06 PM UTC (d0ac589)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/payment-handler/389/f423784...d0ac589.html" title="Last updated on May 10, 2021, 8:06 PM UTC (d0ac589)">Diff</a>

### CODE REVIEW

**Comment by ianbjacobs** [comment]
A note on this pull request: There are some change indications that are not areas that I edited. I'm not sure if this is due to some text not having been tidied before, or new tidy behavior. Sorry for that noise.

**Comment by ianbjacobs** [comment]
More notes:

* If we remove modifiers from PR API, we will want to make corresponding edits here, but that should happen in a separate pull request.
* Also for future consideration depending on changes to PR API: localization of some of the strings (e.g., errors)

**Review by mzkei — APPROVED** [review]

**Review by romandev — APPROVED** [review]
lgtm

**Comment by ianbjacobs** [comment]
@rsolomakhin, does anything need to happen on the Chrome side by virtue of updating the spec this way (e.g., bug tracker as was done for the PR API edit)?

**Comment by rsolomakhin** [comment]
Filed https://crbug.com/1207601.

File: index.html [hunk 1]
```diff
@@ -774,6 +723,13 @@ <h2>
           <pre class="example js" title=
           "Resolving the relative URL of image.src in service worker context.">
 
+
+
+
+
+
+
+
```
Comment (rsolomakhin) [inline index.html]: Nit: What is the purpose of this whitespace?

**Review by rsolomakhin — APPROVED** [review]
LGTM!

File: index.html (hunk 1 above)
Comment (ianbjacobs) [reply index.html]: No reason. I've removed it.

**Pull request merged**